.github/workflows/release.yml

```diff
@@ -26,8 +26,19 @@ jobs:
           cache: npm
           registry-url: https://registry.npmjs.org
 
+      - name: Get package version
+        run: |
+          VERSION=$(npm pkg get version | xargs)
+          echo "Package version is $VERSION"
+          echo "::set-output name=version::$VERSION"
+
       - name: Publish to NPM
         run: npm publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
           NPM_CONFIG_PROVENANCE: true
+
+      - name: Create GitHub Release
+        run: gh release create ${{ steps.get_version.outputs.version }} --title ${{ steps.get_version.outputs.version }} --notes "Full changelog at https://github.com/JedWatson/classnames/blob/main/CHANGELOG.md"
```

Comment:
Thinking that we might want to use the current commit rather than `main`, so that the relevant notes for that release are always on top. WDYT?

Reply:
Sounds good I'll try to add this feature. By the way sorry for the late responses the work has been pretty heavy these few weeks.

Reply:
No worries, I also maintain this stuff in my spare time, so I know the pain :wink:

Reply:
I don't think it is a good idea to use the current commit as if the CHANGELOG did not get updated with that commit it would be empty so I thought linking the main branches CHANGELOG would be a better idea.

Reply:
> I don't think it is a good idea to use the current commit as if the CHANGELOG did not get updated with that commit 

That's ok. I always update the changelog before release. In the future I'd like to automate this proccess also.